config/jobs/kubernetes/sig-network/dualstack-e2e.yaml

```diff
@@ -0,0 +1,75 @@
+presets:
+- labels:
+    preset-cloudprovider-azure-creds: "true"
+  env:
+  - name: AZURE_CREDENTIALS
+    value: /etc/azure-cred/credentials
+  - name: AZURE_SSH_PUBLIC_KEY_FILE
+    value: /etc/azure-ssh/azure-ssh-pub
+  - name: REGISTRY
+    value: k8sprow.azurecr.io
+  - name: KUBE_VERBOSE
+    value: 0
+  volumes:
+  - name: azure-cred
+    secret:
+      secretName: azure-cred
+  - name: azure-ssh
+    secret:
+      secretName: azure-ssh
+  volumeMounts:
+  - name: azure-cred
+    mountPath: /etc/azure-cred
+    readOnly: true
+  - name: azure-ssh
+    mountPath: /etc/azure-ssh
+    readOnly: true
+
+periodics:
+- interval: 6h
+  # ci-dualstack-azure-e2e runs IPv6DualStack feature specific tests on aks-e cluster.
+  name: ci-dualstack-azure-e2e
+  labels:
+    preset-service-account: "true"
+    preset-cloudprovider-azure-creds: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190806-012cf5f-master
+      args:
+      - --job=$(JOB_NAME)
+      - --repo=k8s.io/$(REPO_NAME)=master
+      - --root=/go/src
+      - --service-account=/etc/service-account/service-account.json
+      - --timeout=470
+      - --upload=gs://kubernetes-jenkins/logs/
+      - --scenario=kubernetes_e2e
+      - --
+      - --gce-ssh=
+      - --test=true
+      - --up=true
+      - --down=true
+      - --deployment=acsengine
+      - --build=bazel
+      - --provider=skeleton
+      - --ginkgo-parallel=30
+      - --acsengine-agentpoolcount=2
+      - --acsengine-admin-username=azureuser
+      - --acsengine-creds=$AZURE_CREDENTIALS
+      - --acsengine-orchestratorRelease=1.16
+      - --acsengine-mastervmsize=Standard_DS2_v2
+      - --acsengine-agentvmsize=Standard_D4s_v3
+      - --acsengine-ccm=True
```

Comment:
I dont think you need this as this for buiding the ccm image. https://github.com/kubernetes/test-infra/blob/master/kubetest/azure.go#L64

Reply:
Since you are building from `--repo=k8s.io/$(REPO_NAME)=master`, you probably need to add `--acsengine-hyperkube=True` if you want to build hyperkube from master, instead of using a released version.

Reply:
Thanks for the review @ritazh. Had the changes done locally. Forgot to push the commit. 🤦‍♂

Reply:
Updated!